src/services/script_patcher/script_patcher_service.hpp

```diff
@@ -0,0 +1,93 @@
+#pragma once
+#include <script/scrProgram.hpp>
+#include "memory/pattern.hpp"
+
+namespace big
+{
+	struct script_data
```

Comment:
Should be a class there?

Reply:
I said this because of the `public` access modifier, thinking it was intended, it doesnt matter much